problem_builder/migrations/0004_copy_course_ids.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+def copy_course_id_to_course_key(apps, schema_editor):
```

Comment:
Should we just replace this forward migration with a noop in this `2.6.5.patch1` build, so devops doesn't have to bother manually faking it?